bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -739,7 +771,8 @@ func (r *KubeadmConfigReconciler) joinControlplane(ctx context.Context, scope *S
 		return ctrl.Result{}, err
 	}
 
-	return ctrl.Result{}, nil
+	// Ensure reconciling this object again so we keep refreshing the bootstrap token until it is consumed
+	return ctrl.Result{RequeueAfter: r.TokenTTL / 3}, nil
```

Comment:
nit: given this is using r.TokenTTL / 3 in multiple times I'd put it in a var e.g tokenRequeueAfter that is invoked everywhere

Reply:
done

Reply:
This seems to be still pending